setup.py

```diff
@@ -10,6 +10,8 @@
     'ftw.builder',
     'unittest2',
     'ftw.testbrowser >= 1.26.1',
+    'ftw.testing',
+    'ftw.news'
```

Comment:
@Nachtalb Can we get rid of the ftw.news test dep.?
We could use the plone default news type or our own DX type?